syntax="proto3";

message calc_request {
  int32 var1=1;
  int32 var2=2;
}

message calc_response {
  int32 result=1;
  bool success=2;
}